.env.template

```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY = 
+TRELLO_SECRET_KEY = 
```

Comment:
I think this `TRELLO_SECRET_KEY` is probably a variable you renamed as I can't see it used anywhere, so it would be nice to get rid of that